helpers/helpers.go

```diff
@@ -39,19 +40,14 @@ import (
 	"github.com/pkg/errors"
 )
 
-// PrintError is a utility function to emit an error to the console
-func PrintError(e error) {
-	fmt.Fprintf(os.Stderr, "***Error: %v\n", e)
-}
-
 // CreateCertTemplate will construct the template for needed openssl metadata
 // instead of using an attributes.cnf file
 func CreateCertTemplate() *x509.Certificate {
 	serialNumberLimit := new(big.Int).Lsh(big.NewInt(1), 128)
 	serialnumber, err := rand.Int(rand.Reader, serialNumberLimit)
 	if err != nil {
-		fmt.Println("ERROR: Failed to generate serial number")
-		PrintError(err)
+		log.Println("ERROR: Failed to generate serial number")
```

Comment:
I know this is unrelated to your change, but since you are here can you just print the error on a single line? Much easier when grepping output or log files.